### ISSUE TITLE
Local VideoAlign treats MP4 paths as frame-image lists

### ISSUE THREAD

**Issue by morluto** [Tencent-Hunyuan/UniRL#328]
## Observed behavior

The local VideoAlign scorer assigns the complete `video_paths` list to every Qwen-VL video block. In this message format, a list represents one video supplied as individual frame images; it does not represent multiple video files.

The shipped caller writes generated samples as MP4 files and invokes this path with `batch_size: 1` by default. A value such as `[".../sample_00000.mp4"]` therefore enters the per-frame image loader and fails before VideoAlign produces a score. `VideoAlignRewardScorer` catches that exception and reports the sample as unsuccessful, after which `RewardService` raises on the failure flag.

## Source-derived reproduction

For two paths and prompts, the current method constructs:

```text
prompt-a -> ["video-a.mp4", "video-b.mp4"]
prompt-b -> ["video-a.mp4", "video-b.mp4"]
```

An uncommitted construction probe ran the actual `_VideoAlignInference.reward` method from the base revision and intercepted its input at `_process_vision_info`. This confirms the structure above, but it is not an end-to-end inference reproduction.

The vendored VideoAlign implementation instead pairs `video_paths` and `prompts` with `zip` and gives each video block one string path.

## Expected behavior

Each prompt should receive the path at the same index:

```text
prompt-a -> "video-a.mp4"
prompt-b -> "video-b.mp4"
```

That sends MP4 paths through the video-file decoder and matches the vendored implementation.

## Impact and scope

This input-construction defect affects single-item and multi-item calls. It causes decoding to fail before scoring rather than producing silently mispaired rewards.

Correcting the video block advances execution to later, separate failures in the local scorer's model-call and logits handling. This issue covers only the prompt/video input construction; it does not claim the local scorer becomes functional end to end.
